### DISCUSSION TITLE
[Question]How can I remove the blank space inside the red box?

### DISCUSSION THREAD

**Discussion by LegendaryPistachio** [flet-dev/flet#4413] [Q&A]
### Question

![menu-bar](https://github.com/user-attachments/assets/147d9e67-c3d8-457c-be10-2c7d2c3ab3ee)

I want the top of the menu bar to be close to the bottom of the app bar, without any blank space. How can I remove the blank space inside the red box?

### ------------------------------------------------------

- [X] I have searched for answers to my question both in the issues and in previous discussions.

**Comment by ndonkoHenri** [comment]
Try setting `page.padding=ft.padding.only(top=0)`
([docs](https://flet.dev/docs/reference/types/padding/))

**Reply by LegendaryPistachio** [reply]
verygood！
